llvm/lib/Transforms/Instrumentation/MemProfUse.cpp

```diff
@@ -75,6 +76,17 @@ static cl::opt<unsigned> MinMatchedColdBytePercent(
     "memprof-matching-cold-threshold", cl::init(100), cl::Hidden,
     cl::desc("Min percent of cold bytes matched to hint allocation cold"));
 
+static cl::opt<bool> AnnotationStaticDataPrefix(
+    "annotate-static-data-prefix", cl::init(false), cl::Hidden,
```

Comment:
Add a "memprof-" prefix to the option name to be consistent with the others.

Also I think we can make this default true since we can also control whether the static data profile is generated as part of the memprof profile generation.